deploy/template.yaml

```diff
@@ -143,7 +143,7 @@ Mappings:
       TXMAMESSAGERETENTIONPERIODDAYS: 604800 # Default: 7 days
       GOVUKNOTIFYTEMPLATEID: "bea2a584-4dca-4970-a90d-93977e752fdf"
       CLIENTSESSIONTOKENTTL: 950400 # Default 11 days
```

Comment:
this needs to be removed and YOTISESSIONTTLDAYS to be added, please check of all env vars added/updated

Reply:
Done